README.md

```diff
@@ -14,27 +14,33 @@ In this project, we are setting up a "Hello world" repository with the use of on
 
 [Live Demo Link](http://127.0.0.1:5500/)
```

Comment:
this should be a live demo link of your project. You can do that by clicking on the settings of your branch and then on the left side menu click on pages and configure your live page. If you don't want a live demo, just consider deleting this section.